mysql2.gemspec

```diff
@@ -10,6 +10,13 @@ Mysql2::GEMSPEC = Gem::Specification.new do |s|
   s.homepage = 'https://github.com/brianmario/mysql2'
   s.rdoc_options = ["--charset=UTF-8"]
   s.summary = 'A simple, fast Mysql library for Ruby, binding to libmysql'
+  s.metadata = {
+    'bug_tracker_uri'   => "#{s.homepage}/issues",
+    'changelog_uri'     => "#{s.homepage}/releases/tag/#{s.version}",
+    'documentation_uri' => "https://www.rubydoc.info/gems/mysql2/#{s.version}",
+    'homepage_uri'      => s.homepage,
+    'source_code_uri'   => "#{s.homepage}/tree/#{s.version}",
+  }
```

Comment:
Nit: colon syntax preferred vs. hash-rockets.

Reply:
Unfortunately, you can't use colon syntax when the keys are strings. The Rubygems spec requires the keys to be strings:

>  All keys and values must be Strings

https://guides.rubygems.org/specification-reference/#metadata

Reply:
Oh, well then. Let's merge!

Reply:
Thanks 🎉